core/src/test/java/org/testcontainers/junit/wait/strategy/AbstractWaitStrategyTest.java

```diff
@@ -84,7 +84,11 @@ protected GenericContainer<?> startContainerWithCommand(
      * @param shellCommand the shell command to execute
      */
     protected void waitUntilReadyAndSucceed(String shellCommand) {
-        waitUntilReadyAndSucceed(startContainerWithCommand(shellCommand));
+        try (
+        GenericContainer<?> container = startContainerWithCommand(shellCommand)
+        ) {
```

Comment:
is this correctly formatted? o_O I have never seen Prettier doing it like this 😅

Reply:
build fails because of that 🤔

Reply:
ahhh... yay :D